src/org/scada_lts/dao/MultiChangeHistoryDAOImp.java

```diff
@@ -0,0 +1,94 @@
+import org.scada_lts.dao.model.multichangehistory.MultiChangeHistoryValues;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.util.List;
+
+/**
+ * @author grzegorz.bylica@abilit.eu on 16.10.2019
+ */
+@Repository
+public class MultiChangeHistoryDAOImp implements MultiChangesHistory {
+
+    private static final Log LOG = LogFactory.getLog(MultiChangeHistoryDAOImp.class);
+
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_ID = "mch.id";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_USER_ID = "mch.userId";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_USER_NAME = "mch.username";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_AND_CMP_IDENTIFICATION = "mch.viewAndComponentIdentification";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_INTERPRETED_STATE = "mch.interpretedState";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_TS = "mch.ts";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_VALUE_ID = "vmch.valueId";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_VALUE = "vmch.value";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_DATA_POINT_ID = "vmch.dataPointId";
+    private final static String  COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_XID_POINT = "xidPoint";
+
+
+    // @formatter:off
+	private static final String SQL = "" +
+            "select " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_ID + ", " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_USER_ID + ", " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_USER_NAME + ", " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_AND_CMP_IDENTIFICATION + ", " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_INTERPRETED_STATE + ", " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_TS + ", " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_VALUE_ID + ", " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_VALUE + ", " +
+              COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_DATA_POINT_ID + ", " +
+              "(select xid from dataPoints where id = vmch.dataPointId) as " + COLUMN_NAME_MULTI_CHANGE_HISTORY_VIEW_XID_POINT + " " +
+            "from " +
+                "multi_changes_history mch, " +
+                "values_multi_changes_history vmch " +
+            "where " +
+                "mch.id = vmch.multiChangesHistoryId " +
+            "order by mch.ts desc;";
+
+	// @formatter:on
+    @Override
+    public List<MultiChangeHistoryValues> getHistory(String viewAndCmpId) {
+        if (LOG.isTraceEnabled()) {
+            LOG.trace("SQL UserComents");
+        }
+
+        try {
+
+            List<MultiChangeHistoryValues> listUserComents = DAO.getInstance().getJdbcTemp().query(SQL, (ResultSet rs, int rownumber) -> {
+                    MultiChangeHistoryValues mchv = new MultiChangeHistoryValues();
```

Comment:
Is it possible to change mchv name to more understandable?

Reply:
improved

Reply:
ok